ui-admin/src/generated/search-exp-parser/README.md

```diff
@@ -0,0 +1,4 @@
+This module contains auto-generated TypeScript. Do not edit these files directly.
```

Comment:
```suggestion
This module contains auto-generated TypeScript. They are checked into source so that the frontend can be built and run without Java/gradle dependencies.   Do not edit these files directly.
```